app/views/partials/_search_bar.html.haml

```diff
@@ -2,11 +2,13 @@
   .filters-bar
     .navbar
       .navbar-collapse.collapse
-        %ul.nav.navbar-nav
-          %li
+        .navbar-form
```

Comment:
CSS selector.

That `div` is containing the form created by the `form_tag` I placed later. `text-center` as the name implies is for centering text only.